composer.lock

```diff
@@ -11393,5 +11486,5 @@
     "platform-overrides": {
         "php": "7.2"
     },
-    "plugin-api-version": "1.1.0"
+    "plugin-api-version": "2.0.0"
```

Comment:
Gotcha! You used composer 2 to install the dependencies. Please revert the changes to the composer.json that are not in place, and re-install using composer 1. And finally commit that lockfile in your PR.

Reply:
It seems like this was the only change within the composer.lock file.
The .lock file has been regenerated in the new commit.